src/Item.java

```diff
@@ -1,8 +1,11 @@
 
 public class Item {
+	// class variables
 	private int itemId;
 	private String itemName;
-	private double itemPrepTime;
+	private double itemRate;
+	
```

Comment:
Override the compareTo method as follows 
```suggestion
	@Override
	public int compareTo(Item arg0) {
		return this.getItemId()-arg0.getItemId();
	}
```